txindex/save.go

```diff
@@ -40,3 +40,12 @@ func (db *DB) SaveLastIndexPoint(endBlockHeight int32) error {
 	}
 	return nil
 }
+
+func (db *DB) ClearSavedData(emptyTxPointer interface{}) error {
```

Comment:
`ClearTransactionsIndex` or `ClearSavedTransactions` instead. Clearing saved data should not ordinarily save last index point as 0, but this method does so. It makes more sense if we call this method `ClearSavedTransactions` as it explains why we also set the last index point to 0 after dropping the db.